README.md

```diff
@@ -8,41 +8,54 @@
 
 <img src="https://www.haskell.org/cabal/images/Cabal-light.png" align="right">
 
-This Cabal Git repository contains the following packages:
+This Cabal Git repository contains the following main packages:
 
  * [Cabal](Cabal/README.md): the Cabal library package ([license](Cabal/LICENSE))
  * [Cabal-syntax](Cabal-syntax/README.md): the `.cabal` file format library ([license](Cabal-syntax/LICENSE))
  * [cabal-install](cabal-install/README.md): the package containing the `cabal` tool ([license](cabal-install/LICENSE))
+ * [cabal-install-solver](cabal-install-solve): the package containing the solver component of the `cabal` tool ([license](cabal-install-solver/LICENSE))
 
 The canonical upstream repository is located at
 https://github.com/haskell/cabal.
 
-Installing cabal-install (by downloading the binary)
-----------------------------------------------------
+Ways To Get cabal-install Binary
```

Comment:
Ways to the __the__ cabal-install binary.

Reply:
Good catch. I generally stick to the rule: drop all articles from titles. As I learned from [this english.SE thread](https://english.stackexchange.com/q/38759/155388), this is far from the actual guideline, which, unfortunately, has much less agreement around it than many other stylistic elements (e.g. title capitalization). The gist of the thread is:

1. Omitting articles from titles is merely a widely used practice in media, but (seemingly) not reflected in actual style guides.

2. Articles should be preserved if this decision may change the meaning of the title.

Now, to the title in question. Does the absence of article cheange the meaning? I doubt, but I'm not 100% certain. So, my hunch would be to omit the article here, but again if there are arguments for leaving it in place, or even just strong opinions either side, I'm happy to act on it accordingly.

Reply:
Personally: omit. Let us wait for a native speaker.

Reply:
> drop all articles from titles.

As usual with natural language, it is not that simple.  You can either say "Ways to get cabal-install" or "Ways to get the cabal-install binary", but nothing in between, title or not.  I checked out the examples from the link you posted, and these are in a different class.